holobot/discord/components/dynamic_pager.py

```diff
@@ -94,10 +94,8 @@ async def __run(self):
             except TimeoutError:
                 await message.delete()
                 break
-            except HTTPException as error:
-                # 10008 means the message cannot be found. It was probably deleted by someone.
-                if error.code != 10008:
-                    raise
+            except NotFound: # The message was probably deleted by someone.
```

Comment:
Double check if this actually works.

Reply:
A try-catch was needed for the deletion, too. The original solution handled edit errors only.